.github/workflows/mhs-ci.yml

```
@@ -0,0 +1,63 @@
+name: mhs-ci
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build-mhs-prettyprinter:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: checkout mhs repo
+      uses: actions/checkout@v4
+      with:
+        repository: augustss/MicroHs
+        ref: v0.14.27.0
+        path: mhs
+    - name: make and install mhs
+      run: |
+        cd mhs
+        make minstall
```

```suggestion
    - name: make and install mhs
      run: |
        cd mhs
        make minstall
        echo "$HOME/.mcabal/bin" >> "$GITHUB_PATH"
```
This way, we can avoid having to do `PATH="$HOME/.mcabal/bin:$PATH"` every time.